packages/@react-spectrum/icon/docs/workflow-icons.mdx

```diff
@@ -97,6 +97,6 @@ import LockClosed from '@spectrum-icons/workflow/LockClosed';
 A [searchable list of workflow icons](https://spectrum.adobe.com/page/icons/) is available on the Spectrum website.
 The name of the icon without any whitespace matches the import in React Spectrum.
 
-<!-- ```tsx snippet
+{/* ```tsx snippet
```

Comment:
Should we delete this?

Reply:
it's not currently hurting anything, so I'm ok with it staying for the moment